std/random.d

```diff
@@ -2153,7 +2154,7 @@ if (isFloatingPoint!(CommonType!(T1, T2)) && isUniformRNG!UniformRandomNumberGen
 }
 
 // Implementation of uniform for integral types
-/+ Description of algorithm and suggestion of correctness:
+/** Description of algorithm and suggestion of correctness:
```

Comment:
This comment doesn't look intended for the documentation, and there's simple missing a `/// ditto`